src/SensioLabs/Melody/Tests/IntegrationTest.php

```diff
@@ -118,6 +118,18 @@ public function testRunWithPhpOptions()
         $this->assertContains('memory_limit=42M', $output);
     }
 
+    public function testRunWithForkRepositories()
```

Comment:
Indeed, it'll be better to tests a modified version of Pimple to be sure, that the runned version came from the repository and not the package